.github/workflows/static_build_packaging.yml

```diff
@@ -77,9 +79,118 @@ jobs:
           path: |
             build/tarantool*.deb
             build/tarantool*.rpm
+            build/test-libs*.tgz
           if-no-files-found: error
       - name: Send VK Teams message on failure
         if: failure()
         uses: ./.github/actions/report-job-status
         with:
           bot-token: ${{ secrets.VKTEAMS_BOT_TOKEN }}
+
+  static_build_testing:
+    name: 'static_build_testing (${{ matrix.sys-arch }}) on ${{ matrix.os }}'
+
+    # Run on push to the 'master' and release branches of tarantool/tarantool
+    # or on pull request if the 'full-ci' or 'static-build-ci' label is set.
+    if: github.repository == 'tarantool/tarantool' &&
+        ( github.event_name != 'pull_request' ||
+          contains(github.event.pull_request.labels.*.name, 'full-ci') ||
+          contains(github.event.pull_request.labels.*.name, 'static-build-ci') )
+
+    runs-on: [ self-hosted, Linux, '${{ matrix.sys-arch }}', flavor-8-16 ]
```

Comment:
It doesn't work for aarch64 runners because they have `flavor-8-32` label.

Reply:
fixed